sources/osgUtil/TriangleSphereIntersector.js

```diff
@@ -1,21 +1,26 @@
 'use strict';
 var MACROUTILS = require( 'osg/Utils' );
 var vec3 = require( 'osg/glMatrix' ).vec3;
-var TriangleIntersector = require( 'osgUtil/TriangleIntersector' );
+var TriangleLineSegmentIntersector = require( 'osgUtil/TriangleLineSegmentIntersector' );
 
-var TriangleIntersection = function ( normal, i1, i2, i3 ) {
+// implicitly match with TriangleLineSegment intersection
+var TriangleSphereIntersection = function ( normal, i1, i2, i3, r1, r2, r3 ) {
```

Comment:
I think we just could reuse TriangleIntersection, and not have two different TriangleIntersection containers which basically have exactly the same.